src/navigations/HomeBottomTabs.js

```diff
@@ -113,18 +113,18 @@ function HomeBottomTabs() {
     if (route.name === 'Profile') label = 'Profile';
 
     const bottomBarExtendedStyle = {
-      color: nav?.isFocused() ? activeColor : COLORS.blackgrey
+      color: nav?.isFocused() ? activeColor : COLORS.gray400
     };
     return <Text style={[styles.bottomBarLabel, bottomBarExtendedStyle]}>{label}</Text>;
   };
 
   return (
-    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+    <SafeAreaView style={{flex: 1, backgroundColor: COLORS.almostBlack}}>
       <Loading visible={isLoadingFetchingChannelDetail} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the background color of the SafeAreaView from `COLORS.white` to `COLORS.almostBlack` is a significant visual change. Make sure this change aligns with the overall design language of your application and doesn't negatively impact the visibility of other components within the SafeAreaView.

127:
The change in the inactiveTintColor from `COLORS.blackgrey` to `COLORS.gray400` should be fine as long as it maintains good contrast with the background color for readability purposes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->